### ISSUE TITLE
Add OpenHab 2.0-beta package

### ISSUE THREAD

**Issue by splatch** [openhab/openhab-syno-spk#4]
Would be cool to get `2.0.0-beta 1` package of openhab since it's publicly available. Since new runtime is used it may involve extra work.

**Comment by cniweb** [comment]
Should it be a separate package?
And it should be installed in parallel?

**Comment by splatch** [comment]
I'm fresh user so for me it might be the same package, however I think this may cause troubles for existing users if they could not run OH2 for some reason and will have to revert to previous release. I don't suspect people will run continously two versions at the same time, but ability to test beta version without removing older one would be cool for them

**Comment by yphyph01** [comment]
+1
a separate package would be fine
thank you for your work. Really appreciated

**Comment by krzychb** [comment]
@cniweb

You are enriching life of openHAB users that have a Synology diskstation :smile:

My ranking list is as follows:

1.	Separate package +1
2.	Installed in parallel +1

If this is too much effort I will be happy to try whatever package you are able to prepare :+1:


Krzysztof

**Comment by cniweb** [comment]
OK i add a new brach: https://github.com/cniweb/openhab-syno-spk/tree/2.0-beta1

**Comment by cniweb** [comment]
I add a new prerelease with OpenHab V2.0-beta1: https://github.com/cniweb/openhab-syno-spk/releases/tag/2.0.0-beta1
Testing, please...

**Comment by yphyph01** [comment]
@cniweb 
thank uou to share your work with us.

I've tested the installation of this prerelease  package on dsm 6.0Beat2 and the same issue with the "user home services" occured as on the. 1.8 package.
 Synology has changed some user management commands.
let me know if I can perform additional tests to help you.
regards

**Comment by cniweb** [comment]
A patch for DSM 6.0 is in work, and this version has no patch for user home servce!
Please do not test this version with DSM 6.0!

**Comment by splatch** [comment]
I'm not sure if I'm doing something wrong, cause DSM 5.2 refuses to install this package. When I use older SPKs from https://www.hofrichter.at/sspks/ installation continues and I get wizzard step asking about license acceptance, however when I try to use downloaded package (renamed to from .zip to .spk) I get "invalid file format error" and following entries in `/var/log/messages`:
```
entry.cgi_SYNO.Core.Package.Installation[1].upload[16758]: pkgtool.cpp:1639 Failed to extract package INFO to /volume3/@tmp/56B0EDBD70BFD7AA/
entry.cgi_SYNO.Core.Package.Installation[1].upload[16758]: pkgtool.cpp:1640 /bin/tar xf /volume3/@tmp/upload_tmp.167580 -C /volume3/@tmp/56B0EDBD70BFD7AA/ INFO --no-same-owner
```

**Comment by yphyph01** [comment]
I'm think you don't have to rename the downloaded file. Thé spk is file is archived inside thé downloaded file. simply unzip it.
Yann
Le 3 févr. 2016 12:31 AM, Łukasz Dywicki <notifications@github.com> a écrit :I'm not sure if I'm doing something wrong, cause DSM 5.2 refuses to install this package. When I use older SPKs from https://www.hofrichter.at/sspks/ installation continues and I get wizzard step asking about license acceptance, however when I try to use downloaded package (renamed to from .zip to .spk) I get "invalid file format error" and following entries in /var/log/messages:

entry.cgi_SYNO.Core.Package.Installation[1].upload[16758]: pkgtool.cpp:1639 Failed to extract package INFO to /volume3/@tmp/56B0EDBD70BFD7AA/
entry.cgi_SYNO.Core.Package.Installation[1].upload[16758]: pkgtool.cpp:1640 /bin/tar xf /volume3/@tmp/upload_tmp.167580 -C /volume3/@tmp/56B0EDBD70BFD7AA/ INFO --no-same-owner

**Comment by cniweb** [comment]
Could anyone successfully try the new package already?

**Comment by splatch** [comment]
In my case downloaded zip does not have any spk inside. Spk on this screenshot is renamed zip, as I wrote above:
![openhab spk](https://cloud.githubusercontent.com/assets/199488/12801601/1faf7ff8-cade-11e5-99b7-4c78048d38b5.png)

**Comment by splatch** [comment]
I was able to run SPK. It's possible to launch openhab via shell but launch from package center doesn't work. I was trying to locate any log activity caused by this action but I could not find any. GUI is working too. Great job!

Below a little proof it did work from shell:
![openhub synology](https://cloud.githubusercontent.com/assets/199488/12816957/ac9dcef8-cb4f-11e5-8a76-fd62fe6fb8d3.png)

**Comment by yphyph01** [comment]
Looks the same issue as I described in the issue: https://github.com/cniweb/openhab-syno-spk/issues/3
... No control of the package via the package manager ... and no trace in log files
I'll let you know if any additional findings

**Comment by cniweb** [comment]
I changed the port to 8082! -> https://github.com/cniweb/openhab-syno-spk/blob/2.0-beta1/INFO#L10
Try http://ip-your-ds:8082//openhab.app
Have I missed something perhaps at any point?

**Comment by 7Bit** [comment]
Thanks for your work. Tested the package on Synology DS214Play with DSM 5.2 and Java 8u73. 

Following issues:
* Can't start/stop the via package manager. Starting via start.sh works.
* No log output in /volume1/@appstore/OpenHAB2/logs, only in /volume1/@appstore/OpenHAB2/userdata/logs
* Package is not using my folders at /volume1/public/OpenHab

Port is at 8080 for me, too.

**Comment by cniweb** [comment]
OK, I have the two requirements now separated!
This Issue is only for OpenHab 2.0-beta package < DSM 6

**Comment by xbmcnut** [comment]
@splatch Total OH2 noob question. Installed Beta 2 on DS713+ but package is not running. How do I start using PuTTy?

**Comment by xbmcnut** [comment]
Figured it. `cd /@appstore, sh start.sh`. Working now.

**Comment by cniweb** [comment]
Can I close this issue?

**Comment by yphyph01** [comment]
Hi @cniweb 
Yes, I guees so.
We'll have to test your new package and, if required,to open new issue related to the package.
thank you to share your work 
Regards
